security/cmd/node_agent_k8s/main.go

```diff
@@ -244,21 +248,43 @@ func init() {
 }
 
 func setWorkloadCacheTimeParams() {
+	// Default to 50 milliseconds
+	initialBackoffDuration := 50
+	env, err := strconv.Atoi(os.Getenv(initialBackoff))
+	// if initialBackoff is empty, env will be 0
+	if err == nil && env != 0 {
+		initialBackoffDuration = env
+	} else {
+		log.Errorf("failed to parse initialBackoff field: %v", err)
+		os.Exit(1)
```

Comment:
So if env is unset, the Citadel agent will exit?

Reply:
Can we fall back to 50 if the env is unset? Same for the following.

Reply:
It's a good point, my intention was to detect abnormal setup. If the env is unset, we should use the default value.